### DISCUSSION TITLE
Not getting any vs code suggestions while typing

### DISCUSSION THREAD

**Discussion by maurocig** [vscode-neovim/vscode-neovim#1329] [Q&A]
I wanted to check here before I submit a new issue, has anyone had the same problem?
This came with the latest batch of updates, when I downgrade to 0.096 which is from 5 months ago it works as expected. All new updates from that point just won't show the suggestion panel, unless i manually do it with Ctrl + space.

**Comment by theol0403** [comment]
Works for me. Make sure you aren't recording a macro. If you want to make an issue, check with empty init.vim and also try to bisect the commit if possible.
